### DISCUSSION TITLE
Document where to load Skew corrections

### DISCUSSION THREAD

**Discussion by jcmartinez** [Guilouz/Creality-Helper-Script-Wiki#641] [Features Request]
Klipper documents how to do Skew corrections in software at https://www.klipper3d.org/Skew_Correction.html

Once the Skew profile is saved, it is recommended that it be loaded inside the block [gcode_macro START_PRINT], but this block of code is commented out on my **gcode_macro.cfg** file.

I looked around and found that the [gcode_macro START_PRINT] block that seems to be used in my case resides inside `‎/Helper-Script/KAMP/Start_Print.cfg`, but I don't think I can modify the content of this file because it shows a lock in the Web interface.

What is a good place to load the skew profile by adding a line like this?

``SKEW_PROFILE LOAD=calilantern_skew_profile``

It would be preferred if this line is not erased by future updates.

**Comment by OGMAvision** [comment]
I can confirm that after uninstalling KAMP, the block [gcode_macro START_PRINT] is not commented out and I can load the SKEW calibration profile that I created. If anyone finds a way of having KAMP and loading the SKEW calibration, please document it.

**Comment by SaturnsVoid** [comment]
I added mine in the Machine Start GCode for the printer on OrcaSlicer
![orca-slicer_rTLJ6OcnOK](https://github.com/user-attachments/assets/439764e2-d57b-4ce1-b61e-28eecc643f46)

**Comment by OGMAvision** [comment]
That works, but the skew is a property of the printer, so it should go with the printer. If you sell the printer, lend it, or print files that have been sliced by others, which is a common scenario for a printer at a school, then the slicers won't have the same skew parameters.
